diff --git a/node_modules/@simondotm/nx-firebase/src/executors/build/build.js b/node_modules/@simondotm/nx-firebase/src/executors/build/build.js
index 10235be..d6c7b4b 100644
--- a/node_modules/@simondotm/nx-firebase/src/executors/build/build.js
+++ b/node_modules/@simondotm/nx-firebase/src/executors/build/build.js
@@ -4,7 +4,7 @@ exports.runExecutor = void 0;
 const tslib_1 = require("tslib");
 require("../../utils/e2ePatch"); // intentional side effects
 const devkit_1 = require("@nrwl/devkit");
-const tsc_impl_1 = require("@nrwl/js/src/executors/tsc/tsc.impl");
+const tsc_impl_1 = require("@nx/js/src/executors/tsc/tsc.impl");
 const lib_1 = require("./lib");
 const utils_1 = require("../../utils");
 /**
diff --git a/node_modules/@simondotm/nx-firebase/src/executors/build/lib/get-dependencies.js b/node_modules/@simondotm/nx-firebase/src/executors/build/lib/get-dependencies.js
index b1e94f5..5a38a43 100644
--- a/node_modules/@simondotm/nx-firebase/src/executors/build/lib/get-dependencies.js
+++ b/node_modules/@simondotm/nx-firebase/src/executors/build/lib/get-dependencies.js
@@ -3,7 +3,7 @@ Object.defineProperty(exports, "__esModule", { value: true });
 exports.getFirebaseDependencies = void 0;
 const tslib_1 = require("tslib");
 const devkit_1 = require("@nrwl/devkit");
-const buildable_libs_utils_1 = require("@nrwl/workspace/src/utilities/buildable-libs-utils");
+const buildable_libs_utils_1 = require("@nx/workspace/src/utilities/buildable-libs-utils");
 function getFirebaseDependencies(context) {
     return tslib_1.__awaiter(this, void 0, void 0, function* () {
         devkit_1.logger.log(`- Processing dependencies for firebase functions app '${context.projectName}':`);
